test/get.jwt.token.ts

```diff
@@ -0,0 +1,19 @@
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+
+export class JwtTokenExtractor {
+  public JwtToken: Promise<string>;
```

Comment:
hmm, looking at it again, why is it `Promise<string>`? I think it should be just `string`. In the `getToken` method where we are assigning the value I think `loginResponse.body` should be a `string`?

Reply:
looking at it yet again, I think it should be `private`, and in `getToken` method we should first check if it exists and if so return it, otherwise make a request